controllers/v1/organizationExtension.js

```
@@ -0,0 +1,168 @@
+				} else if (req.method === CONSTANTS.common.API_REQUEST_METHODS.PATCH) {
+					organizationExtension = await organizationExtensionHelper.updateOrgExtension(req)
+				}
+				return resolve(organizationExtension)
+			} catch (error) {
+				return reject({
+					status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+					message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+					errorObject: error,
+				})
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)